src/components/WeatherResults.js

```diff
@@ -23,17 +26,19 @@ function WeatherResults() {
     }
 
     if (isCurrentCityDataLoaded()) {
-      content = `
-       ${UnitSwitch(currentUnits, setCurrentUnits)}
-       <br/>
-       ${WeatherToday()}
-       <br/>
-       ${WeatherForecast()}
-    `;
+      content = (
+        <>
+          <UnitSwitch currentUnits={currentUnits} setCurrentUnitsCB={setCurrentUnits} />
+          <br />
+          <WeatherToday />
+          <br />
+          <WeatherForecast />
+        </>
+      );
     }
   }
 
-  return `<p>${content}</p>`;
+  return <p>{content}</p>;
```

Comment:
I'd make `content = null` by default

Reply:
Good point. Also managed to get rid of ternaries